index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Valentine Jeremiah</h1>
+      <figure id="img-div">
+        <img src="" alt="" id="image">
+        <h3 id="img-caption">This is Jeremiah and Valentine</h3>
+      </figure>
+      <div id="tribute-info">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Unde minus, ratione odio, doloribus quos explicabo similique assumenda culpa doloremque facere est minima iure consequuntur laborum nostrum! Repellendus impedit corporis quibusdam.</div>
```

Comment:
Please wrap your paragraphs with a `p` tag to make your code look more professional.